.circleci/config.yml

```
@@ -350,6 +350,17 @@ jobs:
       - store_artifacts:
           path: /home/circleci/.npm/_logs/
           destination: npm-logs
+      - run:
+          name: 'Create Incident on PagerDuty'
+          when: on_fail
+          command: |
+            curl -X POST --header 'Content-Type: application/json' -d '{
+              "service_key": "${PAGERDUTY_KEY}",
+              "event_type": "trigger",
+              "description": "FAILURE -- React Widgets Tap Test -- ${CIRCLE_BUILD_URL}",
```

Should probably add the build URL as part of the details. Having it as part of the summary would make it too crowded/long.

```suggestion
              "description": "FAILURE: React Widgets TAP Test",
```

---

Yeah that makes sense